### DISCUSSION TITLE
Emojis not working

### DISCUSSION THREAD

**Discussion by sanjeevcoaching** [ImageMagick/ImageMagick#5995] [Development]
### ImageMagick version

3.4.3

### Operating system, version and so on

Abuntu

### Description

I added emojis in input filed and displaying in image emojis not working

**Comment by dlemstra** [comment]
There is nu version 3.4.3 of ImageMagick and I have no idea which input you are talking about? What can we reproduce how?

**Reply by chetan-ids** [reply]
Hi also my colored emoji character not working in imagick image displaying blank & white emojis. Ca you please help?

**Comment by sanjeevcoaching** [comment]
Coloured emojis character not working in Imagick image displaying black & white. How can I add coloured emojis in image?
 - I am using "OpenSansEmoji.ttf" font for add emoji in image.

**Comment by sajalagrawal7** [comment]
Any updates here?
